pkg/storage/tpr/storage_interface_test.go

```diff
@@ -892,6 +894,114 @@ func TestDeleteWithNamespace(t *testing.T) {
 	}
 }
 
+func TestWatch(t *testing.T) {
+	const timeout = 1 * time.Second
+	keyer := getBrokerKeyer()
+	key, err := keyer.Key(request.NewContext(), name)
+	fakeCl := fake.NewRESTClient()
+	iface := getBrokerTPRStorageIFace(t, keyer, fakeCl)
+	resourceVsn := "1234"
+	predicate := storage.SelectionPredicate{}
+	obj := &sc.Broker{
+		TypeMeta:   metav1.TypeMeta{Kind: ServiceBrokerKind.String()},
+		ObjectMeta: metav1.ObjectMeta{Name: name},
+	}
+
+	go func() {
+		if err := fakeCl.Watcher.SendObject(watch.Added, obj, 1*time.Second); err != nil {
+			t.Fatalf("error sending object %#v to watcher (%s)", *obj, err)
+		}
+		fakeCl.Watcher.Close()
```

Comment:
It shouldn't matter because the send is a channel send over an unbuffered `chan`. It will block until the receive happens below (or time out if the receive doesn't happen within 1 second)